README.md

```diff
@@ -8,9 +8,9 @@ The AeroGear Windows cookbook is a list of recipes to quick start your Windows A
 
 | Recipe 	| Core 	| Push 	| Security 	|
 | ------------- |:-------------:| :-----:|:-----:|
-| [HelloWorld](https://github.com/aerogear/aerogear-push-helloworld/ios): Simple demo of Push registration and notification handles with framework as dependencies | - | **push** | - |
+| [HelloWorld](https://github.com/aerogear/aerogear-push-helloworld/tree/master/windows): Simple demo of Push registration and notification handles with framework as dependencies | - | **push** | - |
 | [Shoot](Shoot/README.md): oauth2 sharing photo, web-app to display photos | http, **oauth2** | - | - |

```

Comment:
there is no core, for windows we really have push and oauth2 at this point.

Let me think...